graph4nlp/pytorch/data/dataset.py

```diff
@@ -705,6 +709,41 @@ def _process(self):
             self.test = self.build_topology(self.test)
             if "val" in self.__dict__:
                 self.val = self.build_topology(self.val)
+            # build_edge_vocab and save
+            if self.init_edge_vocab:
```

Comment:
It should be a new function similar to "build_vocab()". E.g., build_edge_vocab()